test/reporter-html-running-class.js

```diff
@@ -0,0 +1,5 @@
+QUnit.module( "Ensure that html reporter adds running class", function() {
```

Comment:
Thanks for adding a test! This, however, should probably go in the existing `reporter-html` test file: https://github.com/qunitjs/qunit/blob/master/test/reporter-html/reporter-html.js

Reply:
@trentmwillis good idea, just added the test there